db/data-generator.js

```diff
@@ -0,0 +1,34 @@
+var faker = require ('faker');
+var fs = require('fs');
+var csvWriter = require('csv-write-stream');
+var writer = csvWriter();
+var generateTracks = require('./tracks-generator.js');
+
+var generateStr = () => {
+  let str = [];
+  for(var i = faker.random.number({min:0, max:9}); i<faker.random.number({min:9, max:16}); i++) {
+    str.push(i);
+  }
+  return str.join(',');
+}
+
+var generateAlbums = () => {
+  console.time('timing seed');
+
+  writer.pipe(fs.createWriteStream('albums.csv'))
```

Comment:
just checking, does this create the file in the right place? in your seed file you have a certain absolute path, does it match up to this? i remember changing my file path in order to make sure the file was created in my db folder instead of root to match up with my seed path